src/components/search/Search.jsx

```diff
@@ -85,6 +95,15 @@ const Search = () => {
   );
   const PAGE_TITLE = `${HEADER_TITLE} - ${enterpriseConfig.name}`;
 
+  // set balance alert values to no-balance if eligible, else low-balance
+  const balanceAlertAdminText = hasNoEnterpriseOffersBalance ? NO_BALANCE_CONTACT_ADMIN_TEXT : LOW_BALANCE_CONTACT_ADMIN_TEXT;
+  const balanceAlertClassName = hasNoEnterpriseOffersBalance ? 'no-offers-balance-alert-with-cta' : 'low-offers-balance-alert-with-cta';
+  const balanceAlertVariant = hasNoEnterpriseOffersBalance ? 'danger' : 'warning';
+  const balanceAlertIcon = WarningFilled;
+  const balanceAlertHeading = hasNoEnterpriseOffersBalance ? NO_BALANCE_ALERT_HEADING : LOW_BALANCE_ALERT_HEADING;
+  const balanceAlertText = hasNoEnterpriseOffersBalance ? NO_BALANCE_ALERT_TEXT : LOW_BALANCE_ALERT_TEXT;
+  const shouldDisplayBalanceAlert = hasNoEnterpriseOffersBalance || hasLowEnterpriseOffersBalance
+
```

Comment:
nit: maybe make a util function in `enterprise-offers/` to do all this stuff?  It's a lot of LOC that have nothing to do with search.